arcgis-ios-sdk-samples/Route & Directions/Find service area interactive/FindServiceAreaInteractive.storyboard

```diff
@@ -1,9 +1,9 @@
 <?xml version="1.0" encoding="UTF-8"?>
-<document type="com.apple.InterfaceBuilder3.CocoaTouch.Storyboard.XIB" version="3.0" toolsVersion="15701" targetRuntime="iOS.CocoaTouch" propertyAccessControl="none" useAutolayout="YES" useTraitCollections="YES" useSafeAreas="YES" colorMatched="YES" initialViewController="uF5-61-Oq4">
+<document type="com.apple.InterfaceBuilder3.CocoaTouch.Storyboard.XIB" version="3.0" toolsVersion="15705" targetRuntime="iOS.CocoaTouch" propertyAccessControl="none" useAutolayout="YES" useTraitCollections="YES" useSafeAreas="YES" colorMatched="YES" initialViewController="uF5-61-Oq4">
```

Comment:
Please revet the changes in this file. They are not related to updating the READMEs.

Reply:
I'm not sure how this change happened..  This happens quite often -- somehow changes like these are made in the storyboard without my knowledge. I usually review the changes before I commit them but they are easily overlooked at times. Do you know why or how I can prevent this from happening?

Reply:
My own suggestion is to avoid touching Storyboards during other development, and do separate commits for them. In this way even if you commit-ed some unwanted changes, it is easier to revert to a previous clean state. 😅 

And often check the diff either locally or on GitHub `Files Changed` tab.

Reply:
@yo1995 I typically don't open the Storyboards during my readme-update process. Sometimes I may select the wrong file on accident but don't purposefully edit anything in that file. So I'm not sure why these Storyboard changes occur. I'm wondering if upon opening the Storyboard, my mouse detects movement and somehow makes changes.

Reply:
My guess is when you load any IB files, storyboards, etc with a later version of Xcode, even if you don't change anything there, Xcode would still change the file version to the latest.   
And any tiny interaction with storyboard will definitely change it.  
Better to discard those changes once you see them 😓